src/generator/AutoRest.Go/TransformerGo.cs

```diff
@@ -136,7 +136,7 @@ private void TransformModelTypes(CodeModelGo cmg)
                     name = $"{name}Type";
                 }
 
-                if (CodeNamerGo.Instance.UserDefinedNames.Contains(name))
+                if (CodeNamerGo.UserDefinedNames.Contains(name))
```

Comment:
Meh, it doesn't really matter I guess. Either way only a single copy should be generated. What are your thoughts, @jhendrixMSFT?